### PR TITLE
Added denominator argument for functions registerDynamic and register…

### PR DESCRIPTION
**Pull request by elvisboghiu**
Added denominator argument for functions registerDynamic and registerAndSetAllRolesDynamic

https://docs.multiversx.com/tokens/nft-tokens#register-dynamic-token
https://docs.multiversx.com/tokens/nft-tokens#register-and-set-all-roles-to-dynamic

### CODE REVIEW

**Review by danielailie — APPROVED** [review]

File: multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py [hunk 1]
```diff
@@ -893,7 +893,7 @@ def create_transaction_for_nft_metadata_recreate(self,
     def create_transaction_for_changing_token_to_dynamic(self,
                                                          sender: IAddress,
                                                          token_identifier: str) -> Transaction:
-        parts: List[str] = [
```
Comment (ssd04) [inline multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py:896]: we can also add an additional check here, we cannot change nft token to dynamic

Comment (ssd04) [reply multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py:896]: we are not able to know token type at this point, but at least add a comment here for this

Comment (popenta) [reply multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py:896]: that is right but we do not have token type as a parameter here.

Comment (popenta) [reply multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py:896]: added docstring

**Review by danielailie — APPROVED** [review]

File: multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py [hunk 2]
```diff
@@ -911,6 +912,7 @@ def create_transaction_for_changing_token_to_dynamic(self,
     def create_transaction_for_updating_token_id(self,
                                                  sender: IAddress,
                                                  token_identifier: str) -> Transaction:
+        """The following token types cannot be changed to dynamic: FungibleESDT, NonFungibleESDT, NonFungibleESDTv2"""
```
Comment (ssd04) [inline multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py:915]: this is not correct here for `updateTokenID`

**Review by ssd04 — CHANGES_REQUESTED** [review]

File: multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py (hunk 2 above)
Comment (popenta) [reply multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py:915]: it's taken from the docs:
https://docs.multiversx.com/tokens/nft-tokens/#update-token

Comment (ssd04) [reply multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py:915]: it is wrong in docs, there is already an open PR for this https://github.com/multiversx/mx-docs/pull/1003/files

**Review by ssd04 — APPROVED** [review]

**Pull request merged**